spec/rubocop/cop/rspec/repeated_example_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::RSpec::RepeatedExample do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense for repeated example' do
+    expect_violation(<<-RUBY)
+      describe 'doing x' do
+        it "does x" do
+                    ^^ Don't repeat examples within an example group.
```

Comment:
I know we can't highlight the entire block contents, but just just highlighting `do` looks a bit strange. Could we highlight the entire line instead?